src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = 0;
+  const DAY_COST = 40;
+  let discount = 0;
+
+  for (let n = 1; n <= days; n++) {
+    if (n >= 7) {
+      discount = 50;
+    }
+
+    if (n < 7) {
+      discount = 20;
```

Comment:
This condition is incorrect. The discount for renting the car for less than 7 days but 3 or more days should only be applied once, not repeatedly in a loop.